packages/next/test/unit/fixtures/01-normalize-routes/vercel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "version": 2,
+  "builds": [
+    {
+      "src": "package.json",
+      "use": "@vercel/next"
+    }
+  ]
+}
```

Comment:
Lets delete this file since its not needed, right?

Reply:
Correct, removing